### ISSUE TITLE
Request: Update for use with FactionsUUID?

### ISSUE THREAD

**Issue by SmallSansSerif** [Jikoo/CreeperHeal#2]
Any way you could update this to play nice with FactionsUUID?

It seems like it only wants the original Mcore-based Factions. 

```
# factions: To use with the Factions plugin. If the settings are set to true,
# the explosions in wilderness or territory are ignored by CreeperHeal (blocks
# are destroyed normally).

factions:
  ignore-wilderness: false
  ignore-territory: true
```

This is the type of error you get: https://pastebin.com/S3pyB3i7

**Comment by Jikoo** [comment]
Semi-possible. FUUID doesn't offer a Maven repo, nor is their repo capable of being compiled by JitPack. These days I refuse to support plugins that can't be obtained via public repo, but I do happen to already have [hacky support for FUUID written](https://github.com/Jikoo/Regionerator/blob/b05bcc064234cb811595ad3fa49ed7ba877ba502/src/main/java/com/github/jikoo/regionerator/hooks/FactionsHook.java).

 Guess I should look at updating CH for 1.17.

**Comment by Jikoo** [comment]
Forgot to tag this issue in the commit. Closed by 163571d83999b2aca6001feced81038c561c0532
